src/pages/profiles/ProfileDetailPanel.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { FC } from "react";
+import { Button, Icon } from "@canonical/react-components";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import Loader from "components/Loader";
+import Aside from "components/Aside";
+import usePanelParams from "util/usePanelParams";
+import { useNotify } from "context/notify";
+import { fetchProfile } from "api/profiles";
+import ProfileLink from "./ProfileLink";
+import { fetchProject } from "api/projects";
+import { isProjectWithProfiles } from "util/projects";
+import ExpandableList from "components/ExpandableList";
+import { isDiskDevice, isNicDevice } from "util/devices";
+import { getProfileInstances } from "util/usedBy";
+import ProfileUsedByProject from "./ProfileUsedByProject";
+
+const ProfileDetailPanel: FC = () => {
+  const notify = useNotify();
+  const panelParams = usePanelParams();
+
+  const profileName = panelParams.profile;
+  const projectName = panelParams.project;
+
+  const {
+    data: profile,
+    error,
+    isLoading: isProfileLoading,
+  } = useQuery({
+    queryKey: [queryKeys.profiles, profileName, projectName],
+    queryFn: () => fetchProfile(profileName ?? "", projectName),
+    enabled: profileName !== null,
```

Comment:
Not sure if we need the enabled flag here. And if we do, why don't we need it below in the project query?